### ISSUE TITLE
Note History

### ISSUE THREAD

**Issue by trymeouteh** [quillpad/quillpad#58]
An option to have note history.

With the option to only store X numbers of older versions of the notes or store older versions of note for X amount of days.

**Comment by bagajohny** [comment]
I would love to have this feature.

**Comment by jsixface** [comment]
Yeah, it's a good feature, simple note has it. But the way I image it to be stored in the filesystem, it is possible to store the history locally within the app. But if you have it synced to either in local folder or in nextcloud, it's outside the scope of the app. It will be whatever the synced backend supports by itself.
